system_tests/logging_.py

```diff
@@ -33,6 +33,32 @@
 TOPIC_NAME = 'gcloud-python-system-testing%s' % (_RESOURCE_ID,)
 
 
+def _retry_backoff(result_predicate, meth, *args, **kw):
+    from grpc.beta.interfaces import StatusCode
```

Comment:
Maybe:  the *pattern* may be more useful than the code:  there are certainly other cases where we can do the backoff to reduce eventual-consistency-driven flakiness.

I've broken my brain a couple of times trying to generalize it, though.